### ISSUE TITLE
Unable to salve custodia Apn

### ISSUE THREAD

**Issue by saltanar** [4IceG/ZTE-MF286D#9]
I can’t save a custom apn , i need everytime time i shutdown the router to send custom apn with at commands, there is way to save them? The settings are already correctly saved on WAN settings.

**Comment by 4IceG** [comment]
Hi @saltanar 

I do it like this:
1. I set apn in LuCI
2. If the modem does not connect to the Internet, I set the APN with the at command
3. Reboot the modem with the at command.
After these operations, my modem has no problems with the connection and remembers the set apn.

**Comment by saltanar** [comment]
The problem is that my modem don’t save apn

Il giorno sab 8 lug 2023 alle 21:45 Rafał Wabik ***@***.***>
ha scritto:

**Comment by 4IceG** [comment]
How do you set the apn? tell me step by step what you do.

**Comment by saltanar** [comment]
I followed your guide, i set the apn on qmi connection, after i send the at
command with the apn, after tha i reboot the modem.
If i shutdown the router i had to do again the procedure.

Il giorno sab 8 lug 2023 alle 21:50 Rafał Wabik ***@***.***>
ha scritto:

**Comment by 4IceG** [comment]
Check if apn is correct (same for QMI and at command).
Try another SIM card (and APN).

It's hard for me to predict what could be wrong, I've never had such a case. 

The procedure has always worked, now I don't even have to force the apn with the at command, the modem immediately connects to the Internet after changing the apn in LuCI.

**Comment by saltanar** [comment]
After some tries the router has saved the settings

 another question, after some times the internet is disconnected and I have
to manually restart the router.
There is a a setting to auto check the internet connection and auto restart
modem and or router  ?



Il giorno sab 8 lug 2023 alle 22:08 Rafał Wabik ***@***.***>
ha scritto:

**Comment by 4IceG** [comment]
There is a package https://github.com/4IceG/luci-app-lite-watchdog, just need to do an update to the latest version.

The package allows you to resume the connection or do a restart of the router / modem.

**Comment by saltanar** [comment]
In your zte firmware there is already this option. I enabled check every 10
minutes and restart router

Il giorno dom 23 lug 2023 alle 14:31 Rafał Wabik ***@***.***>
ha scritto:

**Comment by 4IceG** [comment]
That's right, this package is already in my build only in an older version, so I recommend updating the package.
